.vscode/settings.json

```diff
@@ -5,5 +5,6 @@
     "docs/**/config-options.md": true,
     "docs/**/workspace.md": true,
     "docs/**/type-aliases/*.md": true
-  }
+  },
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
This is necessary so that VS Code uses the TS version to find errors from `node_modules` instead of the VS Code TS version, this saves a lot of time on debugging, because sometimes there are errors due to differences in TS versions, for example when VS Code says that there is an error, but `tsc` in the terminal does not have it